config-proxy/src/main/java/com/yahoo/vespa/config/proxy/RpcConfigSourceClient.java

```diff
@@ -69,7 +69,7 @@
     private Map<ConfigSourceSet, JRTConfigRequester> createRequesterPool(ConfigSourceSet ccs, TimingValues timingValues) {
         Map<ConfigSourceSet, JRTConfigRequester> ret = new HashMap<>();
         if (ccs.getSources().isEmpty()) return ret; // unit test, just skip creating any requester
-        ret.put(ccs, JRTConfigRequester.get(new JRTConnectionPool(ccs), timingValues));
+        ret.put(ccs, new JRTConfigRequester(new JRTConnectionPool(configSourceSet), timingValues));
```

Comment:
How can this build ? ccs -> configSourceSet ?

Reply:
Thanks, fixed